modules/capi-limitations.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-about.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="capi-tech-preview-limitations_{context}"]
+= Cluster API limitations
+
+Using the Cluster API to manage machines is a Technology Preview feature and has the following limitations:
+
+* Only AWS and GCP clusters can use the Cluster API.
+
+* To use this feature, you must enable the `TechPreviewNoUpgrade` feature set.
+Enabling this feature set cannot be undone and prevents minor version updates.
+
+* You must create the primary resources that the Cluster API requires manually.
+For more information, see "Getting started with the Cluster API".
```

Comment:
here should be a hyperlink?

Reply:
Unfortunately, our tooling does not allow `xref` links (from one part of the documentation to another) to originate in any file in the `modules` directory. As a workaround, we state the name of the location where we want to direct the reader, and then include the `xref` in the assembly that includes the module under an `.Additional resources` heading. It is definitely not ideal 🫤 Hopefully, this will be resolved in the medium-term future :crossed_fingers: 

In this case, the workaround looks like this in the output:
![image](https://github.com/openshift/openshift-docs/assets/61474374/17d5661c-664f-433a-bde6-bb73a65c755c)